### DISCUSSION TITLE
【请教】如何获取pag文件里面的layer层的大小和区域？

### DISCUSSION THREAD

**Discussion by leohandsome1688** [Tencent/libpag#2155] [Q&A] [answered]
如题：如何获取pag文件里面的layer层的大小和区域？用了getBounds()方法，但是貌似不好使~~有没有好心人说一下实现的方法或者步骤呀~~愁死了~-

**Comment by kevingpqi123** [comment, marked as answer]
获取文件中的有啥意义，需要获取的大小和区域都是相对于渲染画面，文件的尺寸可以和渲染尺寸不一致，这时会产生缩放和拉伸
https://github.com/Tencent/libpag/discussions/2075
这个帖子中提到的接口建议了解清楚

**Comment by kevingpqi123** [comment]
https://github.com/libpag/pag-ios/blob/master/sample/pag-ios/ViewController/MoveAndRotateLayerViewController.swift
这里有一个 iOS 平台移动文本图层的 demo，使用的是另外一种方法，可以参考下
